format_example.json

```diff
@@ -0,0 +1,104 @@
+{
+  "id": "dfa65597-eb1b-4127-9200-e53cc41d2d93",
+  "name": "Example Project",
+  "summary": "This is a short, one-sentence summary of the Example Project.",
+  "icon": "https://example.com/example.png",
+  "description": "This is a long description describing the project. <b>HTML</b> and **Markdown** can optionally be used.",
+  "gallery": [
+    {
+      "type": "video",
+      "embed": "https://youtube.com/embed...",
+      "caption": "This is an example embed."
+    },
+    {
+      "type": "video",
+      "src": "https://example.com/video.mp4",
+      "caption": "This is an example direct video link."
+    },
+    {
+      "type": "image",
+      "src": "https://example.com/image.png",
+      "caption": "This is an example image."
+    }
+  ],
+  "authors": [
+    {
+      "name": "MCIP Contributors",
+      "uri": "https://github.com/mc-cip"
+    }
```

Comment:
Shall we have a UUID to allow searching by author?

Shall we allow multiple UUIDs so that "cofh team" can link to all their individual profiles as well? Or is that too complicated?

If authors are added or removed is there a way to see which authors wrote a particular version?